food-bank-app/src/screens/News/News.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  View,
+  Text,
+  Image,
+  StyleSheet,
+  SafeAreaView,
+  ScrollView
+} from 'react-native'
+import { NewsCard } from '../../components'
+import { News } from '../../models'
+const testText = 'Una organización sin fines de lucro, constituida en el año 1991 y que tiene como misión “Generar acceso a una alimentación digna para personas en situación vulnerable en nuestra comunidad". Nuestro objetivo es contribuir a la reducción de la inseguridad alimentaria a través de, la entrega de despensas a familias ensituación vulnerable en el Estado de Jalisco.'
+
+const TESTING_NEWS = [
+  {
+    title: '¿Quiénes Somos?',
+    date: new Date(),
+    body: testText,
+    imageUrl: 'https://www.informador.mx/__export/1566991098442/sites/elinformador/img/2019/08/28/rs3243618_dig-2019-08-20-23-33-49_crop1566991097544.jpg_554688468.jpg'
+  },
+  {
+    title: 'Nuevo centro de acopio',
+    date: new Date(),
+    body: testText,
+    imageUrl: 'https://www.informador.mx/__export/1566991098442/sites/elinformador/img/2019/08/28/rs3243618_dig-2019-08-20-23-33-49_crop1566991097544.jpg_554688468.jpg'
+  },
+  {
+    title: 'El reto: acabar con el hambre',
+    date: new Date(),
+    body: testText
+  }
+]
+
+const Header = (): JSX.Element => {
```

Comment:
Is another component necessary?

Reply:
Not really, it is just for readability, i could change it though. What you think?

Reply:
Again, this is pure style, but [AirBnB convention](https://github.com/airbnb/javascript/blob/master/react/README.md) says one component per file:

> Only include one React component per file.

It's up to you, I don't think this is a reason to slow down development

Reply:
I corrected this